### DISCUSSION TITLE
Download failed

### DISCUSSION THREAD

**Discussion by edvantage24** [aclap-dev/video-downloadhelper#723] [Help and Q&A]
### Your question

Download failed intermittently on a website. Kindly investigate and provide fix, thanks!

### Addon & CoApp version and details

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://www.viu.com/ott/hk/zh/vod/428973/%E5%BF%98%E4%B8%8D%E4%BA%86",
  "error": {
    "details": "SideDownload error: : [tls @ 0x7fbc55b5ce80] error:0A000172:SSL routines::wrong signature type\n[hls @ 0x7fbc5580aa00] Unable to open key file https://wstreamks.now.com/hls/vodcp57/vodcp57.key\n[hls @ 0x7fbc5580aa00] Error when loading first segment 'https://ewcdn109.nowe.com/session/p9-53ae99dfe2b25efaa8d17a984abca/hls/vodcp57/201903070742406/201903070742406_Layer4/6752_Period1/segment0.ts'\n[in#0 @ 0x7fbc5580a8c0] Error opening input: Invalid data found when processing input\nError opening input file https://ewcdn109.nowe.com/session/p9-53ae99dfe2b25efaa8d17a984abca/hls/vodcp57/201903070742406/201903070742406_Layer4_vod.m3u8?token=b7055786c84f3ae8591dc83580a19701_1716863061.\nError opening input files: Invalid data found when processing input\n",
    "downloadable_id": "downloadable_4121075683730387",
    "error": "coapp_failure",
    "id": "coapp_failure_ac7cb7bb-a51e-4819-b9b2-c3bf017e0928"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

The URL in the page_url tag in your problem report automatically transfers me to this URL:

https://www.viu.com/ott/no-service/

I suspect you have to be a member, perhaps a paying member, of the web site, in order to view the web page you are dealing with.  I am not such a member & have no intention of becoming one.  I also will not accept your login credentials, even temporarily.  You may have a lot of trouble getting help because of this.
